.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,45 @@
+### Modify this title
+
+Related Issue/Asana ticket: \_
+
+Short description: \_
+
+<!--
+Examples:
+
+Title: Add Search Functionality to User Dashboard.
+Related Issue: #1234
```

Comment:
Does Asana ticket have number? I didn't find it (except the id that is changing in the url)

Reply:
No it doesn't. My mistake. I've updated the examples